doc/source/changes/version_0_28.rst.inc

```diff
@@ -98,12 +98,55 @@ New features
     >>> arr1.equals(arr2, nan_equal=True)
     True
 
+ This method also includes the arguments `rtol` (relative tolerance) and `atol` (absolute tolerance)
+ allowing to test the equality between two arrays within a given relative or absolute tolerance:
+
+    >>> arr1 = LArray([6., 8.], "a=a0,a1")
+    >>> arr1
+    a   a0   a1
+       6.0  8.0
+    >>> arr2 = LArray([5.999, 8.001], "a=a0,a1")
+    >>> arr2
+    a     a0     a1
+       5.999  8.001
+    >>> arr1.equals(arr2)
+    False
+    >>> # equals returns True if abs(array1 - array2) <= (atol + rtol * abs(array2))
+    >>> arr1.equals(arr2, atol=0.01)
+    True
+    >>> arr1.equals(arr2, rtol=0.01)
+    True
+
+  Closes :issue:`488` and :issue:`518`.
+
+* added `Load from Script` in the File menu of the editor allowing to load commands from an
+  existing Python file (closes :editor_issue:`96`).
+
+* added `Edit` menu allowing to undo and redo changes of array values by editing cells and
+  removed `Apply` and `Discard` buttons. Changes are now kept when switching from an array to another
+  instead of losing them as previously (closes :editor_issue:`32`).
+
+* allowed to provide an absolute or relative tolerance value when comparing arrays through the `compare` function
+  (closes :editor_issue:`131`).
+
+* made the editor able to detect and display plot objects stored in tuple, list or arrays.
+  For example, arrays of plot objects are returned when using `subplots=True` option in calls of `plot` method:
+
+    >>> a = ndtest('sex=M,F;year=2000..2017')
+    >>> p = a.plot(subplots=True)
+    >>> type(p)
```

Comment:
remove the next 4 lines? our users do not know/care what this is about.

Reply:
Honestly, my idea was to use the changelog to also teach some little Python tricks to our users but this is maybe pointless and not the right place.

Reply:
I indeed don't think this is the right place for that, but well leave that in if you want, I don't really care 😉